vernac/prettyp.ml

```diff
@@ -436,14 +436,14 @@ let pr_located_qualid = function
   | Abbreviation kn ->
       str "Notation" ++ spc () ++ pr_path (Nametab.path_of_abbreviation kn)
   | Dir dir ->
-      let s,dir =
+      let s,mp =
         let open Nametab in
         let open GlobDirRef in match dir with
-        | DirOpenModule { obj_dir ; _ } -> "Open Module", obj_dir
-        | DirOpenModtype { obj_dir ; _ } -> "Open Module Type", obj_dir
-        | DirOpenSection { obj_dir ; _ } -> "Open Section", obj_dir
+        | DirOpenModule mp -> "Open Module", mp
+        | DirOpenModtype mp -> "Open Module Type", mp
+        | DirOpenSection _ -> anomaly (Pp.str"Sections are not locatable")
```

Comment:
Maybe we should make them locatable instead?

Reply:
We could, but that's another direction of work. Until we have that feature (if desirable), I prefer that the types reflect the current situation.

Reply:
Not sure what this has to do with the types, if sections are locatable we would to_string the dirpath so the types in this PR work fine.